docs/userguide/datafiles.rst

```diff
@@ -356,24 +356,21 @@ even if they were listed in ``package_data`` or were included as a result of usi
 Interplay between these keywords
 --------------------------------
 
-TODO:
-- better formatted logic expression
-- check logic correctness (especially regarding parentheses)
-
 Meanwhile, to further clarify the interplay between these three keywords,
 to include certain data file into the source distribution, the following
 logic condition has two be met::
 
-    m or (p and not (i or e))  # TODO: why not i?
+    m or (p and not e)
```

Comment:
I think for documentation, using plain text (or, and, not) might be more readable than symbols.

Also perhaps we should link to [the original experiment repo](https://github.com/abravalheri/experiment-setuptools-package-data?tab=readme-ov-file#results-for-setuptools6060) in case someone want to know different behaviour in different versions of `setuptools`

Reply:
If we are going for plain text, then it is probably better to use the full keywords:

```
MANIFEST.in OR (package-data AND NOT exclude-package-data)
```

Or something like that.

Reply:
> If we are going for plain text, then it is probably better to use the full keywords

This one looks great! It's much better to avoid defining such custom notations for simplicity.

Meanwhile it looks like all capital keywords (AND/OR/NOT) would no be highlighted while and/or/not will, it is just my browser?

Reply:
Well, please fell free to highlight or use any kind of emphasis (e.g. bold, italics, uppercase, etc) the way you think makes more sense 😅

Reply:
Of course, thanks for the advice!